site/components/TransactionsModal.js

```diff
@@ -0,0 +1,146 @@
+            <SvgContainer name="close" />
+          </button>
+          <p className="mb-2 font-bold text-left">
+            {t(`${transaction.operation}`)}
+          </p>
+        </div>
+        <div className="mt-4">
+          {/* Values sent and received */}
+          {transaction.sent && (
+            <div className="flex items-center justify-between pb-2 text-lg font-bold border-b border-gray-300">
+              <div className="w-2/5 text-left">
+                {`${formatNumber(transaction.sent)} ${transaction.sentSymbol}`}
+              </div>
+              <div className="w-1/5 text-2xl">→</div>
+              <div className="w-2/5 text-right">
+                {transaction.received.map((token, i) => (
+                  <div key={i}>{`${formatNumber(token.value)} ${
+                    token.symbol
+                  }`}</div>
+                ))}
+              </div>
+            </div>
+          )}
+
+          {/* General operation information: total txs, fees, status */}
+          <div className="py-4">
+            <TransactionModalRow
+              text={t('total-transactions')}
+              value={transaction.suffixes && transaction.suffixes.length}
+            />
+            <TransactionModalRow
+              text={isConfirmed ? t('total-tx-fee') : t('estimated-tx-fee')}
+              value={`${formatNumber(
+                isConfirmed ? transaction.actualFee : transaction.expectedFee
+              )} ETH`}
+            />
+            <JustifiedBetweenRow
+              keyComponent={
+                <p className="text-sm text-gray-350">{t('global-tx-status')}</p>
+              }
+              valueComponent={
+                <p className="text-sm font-semibold">
+                  {t(`status-${transaction.transactionStatus}`)}
+                </p>
+              }
+            />
+          </div>
+
+          {/* Transactions list and status: name, number, status, hash */}
+          {transaction.suffixes.map((suffix, idx) => (
+            <div className="py-4 border-t border-gray-300" key={suffix}>
+              <TransactionModalRow
+                text={`${t('transaction')}: ${t(suffix)}`}
+                value={`${idx + 1}/${transaction.suffixes.length}`}
+              />
+              <TransactionModalRow
+                text={t('status')}
+                value={
+                  transaction[`transactionStatus-${idx}`]
+                    ? t(`status-${transaction[`transactionStatus-${idx}`]}`)
+                    : t('status-waiting-wallet')
+                }
+              />
+              {transaction[`transactionHash-${idx}`] && (
+                <>
+                  <TransactionModalRow
+                    text={t('transaction-hash')}
+                    value={
+                      <EtherscanLink
+                        tx={transaction[`transactionHash-${idx}`]}
+                      />
+                    }
+                  />
+                </>
```

Comment:
This Fragment can be dropped.

```suggestion

                  <TransactionModalRow
                    text={t('transaction-hash')}
                    value={
                      <EtherscanLink
                        tx={transaction[`transactionHash-${idx}`]}
                      />
                    }
                  />
```

Reply:
Changed in b73e76b.